package.json

```diff
@@ -10,6 +10,7 @@
     "grunt": "0.4.5",
     "grunt-angular-templates": "0.5.7",
     "grunt-available-tasks": "0.4.3",
+    "grunt-cli" : "~0.1.13",
```

Comment:
Is this required because we cannot install `grunt-cli` globally on the openshift box?  Is there a way to install this by configuring openshift appropriately?